tests/snappi/pfcwd/files/pfcwd_burst_storm_helper.py

```diff
@@ -220,7 +238,7 @@ def __gen_traffic(testbed_config,
         pause_pkt.pause_class_6.value = pause_time[6]
         pause_pkt.pause_class_7.value = pause_time[7]
 
-        pause_flow_start_time = id * (pause_flow_dur_sec + pause_flow_gap_sec)
+        pause_flow_start_time = id * (pause_flow_dur_sec + pause_flow_gap_sec) + WARM_UP_TRAFFIC_DUR
```

Comment:
Maybe we should introduce a new argument "pause_flow_start_delay_sec" for __gen_traffic function

Reply:
The id value is generated within the function itself (as well as pause_flow_dur_sec + pause_flow_gap_sec), so I prefer to keep it like it is for now. We can rename the variable to pause_flow_start_delay_sec?